apps/remix-ide-e2e/src/tests/libraryDeployment.test.ts

```diff
@@ -100,14 +100,14 @@ function checkDeployShouldSucceed (browser: NightwatchBrowser, address: string,
     .clickLaunchIcon('udapp')
     .selectContract('test') // deploy lib
     .createContract('')
-    .getAddressAtPosition(1, (address) => {
+    .getAddressAtPosition(0, (address) => {
       addressRef = address
     })
-    .waitForElementPresent('.instance:nth-of-type(3)')
-    .click('.instance:nth-of-type(3) > div > button')
+    .waitForElementPresent('.instance')
+    .click('.instance > div > button')
     .perform(() => {
       browser
-        .testConstantFunction(addressRef, 'get - call', null, '0:\nuint256: 45')
+        .testConstantFunction(addressRef, 'getInt - call', null, '0:\nuint256: 45')
```

Comment:
calling  `get` should have worked. 
`getInt` is calling the lib, but here we are testing that the lib is correctly linked to the contract.
the contract is:
```
function get () public view returns (uint) {
          return lib.getInt();
      }
```

Reply:
@yann300 I will need your help to figure out why testing `get` from libs fail.